torch_geometric/nn/conv/han_conv.py

```diff
@@ -11,8 +11,8 @@
 from torch_geometric.utils import softmax
 
 
-def group(xs: List[Tensor], q: nn.Parameter,
-          k_lin: nn.Module) -> Optional[Tensor]:
+def group(xs: List[Tensor], q: nn.Parameter, k_lin: nn.Module,
+          return_semantic_attention_weights=None):

```

Comment:
Shall we rename to `return_attention_weights`?

Reply:
Thank you for your review! HAN has two level attention mechanism 1) a node-level one that represents how much the features of the nodes in the neighborhood are important for a certain node 2) a semantic-level attention which can be interpreted as the contribution of a certain metapath for a specific task. I think that the renaming to simple return_attention_weights could be too generic and lead to potential usage errors.

Reply:
Ok, let's keep the name to avoid confusion :)